bin/log-conventions.js

```diff
@@ -0,0 +1,57 @@
+require('colors')
+
+function formattedOutput (location, lines) {
+  console.log(
+    `${'Console log function'.red} in ${location.white}
+
+${lines}
+
+Consider using our logger (log.debug) or add this line ${'/* ignore-console-log */'.green} above to add an exception
+${'------------------------------------------------'.blue}`
+  )
+}
+
+const walk = require('walk')
+const fs = require('fs')
+const path = require('path')
+const walker = walk.walk('./src')
+let found = 0
+
+walker.on('file', function (root, fileStats, next) {
+  if (!fileStats.name.includes('.js')) next()
+  const filename = path.join(root, fileStats.name)
+  fs.readFile(filename, 'utf-8', function (err, data) {
+    if (err) throw err
+    const lines = data.split('\n')
+    for (let i = 0; i < lines.length; i++) {
+      const line = lines[i]
+      if (line.indexOf('console.') !== -1) {
```

Comment:
I don't really like the right drift here, more a coding style thing. Will try to reimplement it a bit nicer.